book-crawler/app/crawler/kyobo.py

```
@@ -7,21 +7,61 @@
+        if any(keyword in page_source.lower() for keyword in ["blocked", "captcha", "접근", "차단", "오류", "error", "403", "봇"]):
+            logger.warning(f"차단 패턴 감지 - Title: {page_title}, URL: {current_url}")
+            
+            # 스크린샷 저장 (디버깅용)
+            screenshot_path = f"blocked_{url.split('/')[-1]}.png"
+            driver.save_screenshot(screenshot_path)
+            logger.info(f"차단 페이지 스크린샷 저장: {screenshot_path}")
+            
+            # 페이지 소스 일부 저장
+            with open(f"blocked_{url.split('/')[-1]}.html", "w", encoding="utf-8") as f:
+                f.write(driver.page_source[:2000])
+            logger.info(f"차단 페이지 HTML 저장")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)